www/features/on.md

```diff
@@ -20,7 +19,10 @@ on [every] <event-name>[(<param-list>)][\[<filter>\]] <count> [<debounce> | <thr
 
 If the `every` prefix is used, the event handler will not be synchronized (see [queueing](#queueing) below.)
 
-The `event-name` can be a symbol, a dot-separated symbol or a string that names the event to respond to
+The `event-name` can be a symbol, a dot-separated symbol or a string that names the event. The most obvious events
```

Comment:
One of the primary reasons for hyperscript is to make convenient event handlers for custom events, too, like those triggered by [htmx](https://htmx.org/reference/#events) or other javascript libraries.  I'd suggest mentioning this even before the standard browser events, which already have their own dedicated attributes.

Reply:
Can you clarify? My understanding is that events like click are not specifically in hyperscript, but are basically just the standard browser events? e.g. does hyperscript have a special understanding of "click", or is it just the browser event with possibly some syntactic sugar?

Custom (synthetic? higher level?) events probably should get their own section.

FWIW for new users I suspect that the browser events are the most interesting place to start (look! I clicked and made a thing happen!) and then once someone gets a little further along the higher level events start to become more compelling.

Reply:
I think what Ben means is that people won't be interested in handling browser events, as they can already use `onclick="..."` with JS. Of course hyperscript has more enticing features, but we haven't convinced people to read about them. Using hyperscript as a "library glue" is a clear use case that can get people interested to learn more..